src/pages/Hojung/Main/Main.scss

```
@@ -0,0 +1,593 @@
+@mixin flex-align-center {
+  display: flex;
+  align-items: center;
+}
+
+img {
+  max-width: 100%;
+  vertical-align: top !important;
+}
+
+* {
+  font-size: 14px;
+}
+
+/* nav */
+#header {
+  border-bottom: 1px solid #ddd;
+  nav {
+    height: 55px;
+    @include flex-align-center;
+    justify-content: space-between;
+    .logoZone {
+      font-family: 'Lobster', cursive;
+      font-size: 20px;
+      line-height: 1.3em;
+      font-weight: 400;
+    }
+    .searchZone {
+      position: relative;
+      .searchPop {
+        display: none;
```

```suggestion
.header {
  border-bottom: 1px solid #ddd;
  
  nav {
    height: 55px;
    @include flex-align-center;
    justify-content: space-between;
    
    .logoZone {
      font-family: 'Lobster', cursive;
      font-size: 20px;
      line-height: 1.3em;
      font-weight: 400;
    }
    
    .searchZone {
      position: relative;
      
      .searchPop {
        display: none;
```
- 여기도 한줄씩 띄워서 가독성 확보해주시고 id 셀렉터 쓸지 클래스 셀렉터 쓸지도 다시 검토해주세요
- 바깥쪽 요소의 높이를 고정값으로 주지 말고 자식요소의 합(자식 높이 + Margin, padding)으로 부모요소의 높이가 결정될 수 있도록 해주세요

---

수정완료 했습니다